app/lib/project/MetapolatorProject.js

```diff
@@ -655,50 +645,56 @@ define([
                                this.fontinfoFileName, this.fontinfoFile );
     };
 
-    function exportInstance(io, project, masterName, dirName, precision) {
-        // returns a models/Controller
-        var model = project.open(masterName)
-          , master = model.query('master#' + masterName)
-          , glyphSet
-          , exportController
-          ;
-
-        // create a bare ufoV2 directory
-        io.mkDir(false, dirName);
-
-        // create dirName/metainfo.plist
-        io.writeFile(false, dirName+'/metainfo.plist'
-                                , plistLib.createPlistString(metainfoV2));
-
-        // fontforge requires a fontinfo.plist that defines unitsPerEm
-        io.writeFile(false, dirName+'/fontinfo.plist'
-                                , plistLib.createPlistString(minimalFontinfo));
+    _p.exportUFOInstance_chunk = function(masterName, dirName, precision) {
+        if (this.chunk_io[masterName] == undefined){
+            this.chunk_io[masterName] = new InMemory();
+        }
 
-        io.mkDir(false, dirName+'/glyphs');
-        io.writeFile(false, dirName+'/glyphs/contents.plist', plistLib.createPlistString({}));
+        if (this.chunk_exportController[masterName] == undefined){
+            this.chunk_exportController[masterName] = new UFOExportController(this.chunk_io[masterName], this, masterName, dirName, precision);
+        }
 
-        glyphSet = GlyphSet.factory(false, io, dirName+'/glyphs', undefined, 2);
+        var it = this.chunk_exportController[masterName].run_export_iteration();
+        if (it.done){
+            delete this.chunk_exportController[masterName];
+        }
+        return it;
+    };
 
-        exportController = new UFOExportController(master, model, glyphSet, precision);
-        exportController.do_export();
+    function exportUFOInstance(io, project, masterName, dirName, precision) {
```

Comment:
With the changes in `UFOExportController` you can undo the changes here. Don't forget that `do_export` should become `doExport` on the way.